operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServerUpIteratorStepTest.java

```diff
@@ -293,6 +293,27 @@ public void maxClusterConcurrentStartup_doesNotApplyToNonClusteredServers() {
     assertThat(MS3 + " pod", domainPresenceInfo.getServerPod(MS3), notNullValue());
   }
 
+  @Test
+  public void whenClusteredServersAlreadyScheduled_canStartNonclusteredServer() {
```

Comment:
Since the issue we are fixing here involves addin 2 non-clustered managed servers, should we add a test case where 2 non-clustered managed servers are added and make sure that both will be started?

Reply:
I believe that `maxClusterConcurrentStartup_doesNotApplyToNonClusteredServers` covers that case.

Reply:
that test doesn't seem to verify MS4 will be started.

Reply:
You're correct. I am going to do some cleanup of the tests in that class. There is more going on than is obvious.

Reply:
The revised test is `nonClusteredServers_ignoreConcurrencyLimit`